server/svix-server/src/lib.rs

```diff
@@ -65,15 +71,15 @@ pub async fn run(cfg: Configuration, listener: Option<TcpListener>) {
     let with_api = cfg.api_enabled;
     let with_worker = cfg.worker_enabled;
 
-    let listen_address = SocketAddr::from_str(&cfg.listen_address).unwrap();
-
+    let listen_address =
+        SocketAddr::from_str(&cfg.listen_address).expect("Error coercing http address");
     let (server, worker_loop) = tokio::join!(
         async {
             if with_api {
                 if let Some(l) = listener {
                     tracing::debug!("API: Listening on {}", l.local_addr().unwrap());
                     axum::Server::from_tcp(l)
-                        .unwrap()
+                        .expect("Error starting http server")
```

Comment:
Just to confirm, does unwrap() hide the actual underlying error, or just supplement it?

Reply:
Cool, perfect!